test/k8sT/manifests/v1.5/cilium-ds-patch.yaml

```diff
@@ -15,6 +15,7 @@ spec:
         - "--pprof=true"
         - "--log-system-load"
         - "--config-dir=/tmp/cilium/config-map"
+        - "--debug-verbose=flow"
```

Comment:
Does this work with the helm stuff now? Also, this is the manifest for v1.5 but AFAICT you're not doing the upgrade tests so it should be running the dev/master version of the test..?

Reply:
Please ignore this patch leftover.